src/chrome/content/rules/uniongang.tv.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+mail.uniongang.tv ⁴
+uniongang.tv mixed content
+www.uniongang.tv mixed content
+
+
+⁴ self signed
+-->
+<ruleset name="uniongang.tv (partial)">
+	<target host="forum.uniongang.tv" />
```

Comment:
This has also mixed content. Could you add the `platform` and the other MCB targets?